File: lab/javascript/matematica_e_operadores/maioridade.js
function maioridade(idade){
    if (idade < 0){
        throw new Error("Idade inválida");
    }
    return idade >= 18 ? "Maior de idade" : "Menor de idade";
}


// Testes
let lista = [0, 5, 17, 18, 20, 99];
for (let i=0; i < lista.length; i++){
    console.log(`${lista[i]}: ${maioridade(lista[i])}`);
}

File: lab/python/matematica_e_operadores/maioridade.py
def maioridade(idade: int) -> str:
    if idade < 0:
        raise ValueError("Idade inválida")
    return "Maior de idade" if idade >= 18 else "Menor de idade"


# Testes
for idade in [0, 5, 17, 18, 20, 99]:
    print(f"{idade}: {maioridade(idade)}")
